app/models/lesson.rb

```diff
@@ -21,6 +22,16 @@ def position_in_section
     ).count
   end
 
+  # Obtains lesson content from GitHub and saves it to the database
+  def import_content
```

Comment:
there's a lot going on in this method, can we refactor it out so it has a single responsibility. something like this please: